### ISSUE TITLE
Error when solution file is not in the root of the git repository

### ISSUE THREAD

**Issue by rvdginste** [GitTools/GitLink#99]
This is based on the develop branch of the repo (commit 93237bbfbca506a0211f8f5104075ea8af6afb77).

When I run gitlink I get an error because (as I understand) the Git repository is initialized with the directory where the solution file is located. This gives an error because the constructor of the Repository class expects the location  of the ".git" folder, or the root of the repository.

**Comment by GeertvanHorrik** [comment]
On what system are you running this command and what command line parameters have you specified? I think GitLink either creates a dynamic repository (in the temp directory) or uses an existing .git directory in the solution root (not necessarily the .sln-file directory).

**Comment by rvdginste** [comment]
I actually found GitlinkTask in the new prerelases and wanted to use that. That gave an error and then I tried to run it from Visual Studio using the same parameters that the MSBuild task would use.
`GitLink.exe c:\development\X.Server\src -include X.Server -c Debug`

The c:\development\X.Server folder is the root folder of the git repository. The solution is placed in X.Server\src\X.Server.sln. It tries to use the given "c:\development\X.Server\src" folder as the root for the git repository.
